js/docx/smoke/api_table_row_pr/to_json.js

```diff
@@ -0,0 +1,27 @@
+builder.CreateFile("docx");
+    let oDocument = Api.GetDocument();
+    let oParagraph = oDocument.GetElement(0);
+    let oTableStyle = oDocument.CreateStyle("CustomTableStyle", "table");
+        oTableStyle.SetName("My Custom Table Style");
+    let oTableRowPr = oTableStyle.GetTableRowPr();
+        oTableRowPr.SetHeight("atLeast", 720);
+    let jsonTableStyle = oTableStyle.ToJSON();
+        GlobalVariable["JSON_TableStyle"] = jsonTableStyle;
+        oParagraph.AddText(jsonTableStyle);
+builder.CloseFile();
+
+//////////////
+
+builder.CreateFile("docx");
+    let oDocument = Api.GetDocument();
+    let oParagraph = oDocument.GetElement(0);
```

Comment:
For what purpose do you refer to this element?

Reply:
Fix in the new commit.